uint8 index
---
bool state
